examples/hello-world-http/registerUnregisterHTTPExample.js

```diff
@@ -1,21 +1,21 @@
-const {HttpService, Reshuffle} = require('../..');
+const { HttpConnector, Reshuffle} = require('../..');
```

Comment:
Do you think in all our examples we should consistently import `Reshuffle` first? So 
```suggestion
const { Reshuffle, HttpConnector} = require('../..');
```

Reply:
I think we should format code in examples. I'll change it to do so and also add a rule to sort imports out so it's consistent.